From 6efc12651d25c7a368b87487839750114de89436 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tomasz=20=C5=9Awi=C4=85tek?=
 <57900186+stpf99@users.noreply.github.com>
Date: Thu, 21 Sep 2023 17:13:00 +0200
Subject: [PATCH] hwdb: Bush tablet rotation support (#29268)

(cherry picked from commit eb5e1fe7e87ffa25d9a67e9b5538ccfb9de901c9)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 116437e77b..e78c795241 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -209,6 +209,14 @@ sensor:modalias:acpi:KIOX010A:*:dmi:*:svnAMI:*:skuH2M6:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, -1, 0; 0, 0, -1
  ACCEL_LOCATION=display
 
+#########################################
+# BUSH
+#########################################
+
+# Bush Windows tablet
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnBUSH*:pnBushWindowstablet:*
+ ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 1, 0, 0
+
 #########################################
 # Chuwi
 #########################################
